opentreemap/importer/models/base.py

```diff
@@ -28,7 +28,7 @@ class Meta:
     CREATING = 4
     FINISHED_CREATING = 5
     FAILED_FILE_VERIFICATION = 6
-    CANCELED = 7
+    CANCELED = 8
```

Comment:
Consider moving `LOADING` to be above cancelled, so all of the statuses are in numerical order, which should hopefully prevent this from happening again in the future